prac_07/myguitars.py

```diff
@@ -6,20 +6,68 @@
 """
 import csv
 from guitar import Guitar
-from operator import itemgetter
 
 
 def main():
-    guitars = []
+    guitars = []  # set guitars to the empty list
+    open_file(guitars)
+    name = input("New guitar name: ")
+    get_new_guitar(guitars, name)  # create Guitar object
+    guitars.sort()
+    print_guitars(guitars)
+    write_file(guitars)
+
+
+def write_file(guitars):
+    """Write list of files to csv"""
+    with open("guitars.csv", "w") as out_file:
+        writer = csv.writer(out_file)
+        for guitar in guitars:
+            writer.writerow(guitar)
+
+
+def print_guitars(guitars):
+    """Print every guitar in list"""
+    for guitar in guitars:
+        print(guitar)
+
+
+def get_new_guitar(guitars, name):
+    """Create new Guitar object"""
+    while name != "":
+        guitar = get_guitar(name)  # create Guitar object
+        guitars.append(guitar)  # add to existing list of guitars
+        print_added_guitar(guitar)  # added guitar message
+        name = input("Name: ")
```

Comment:
Instead of using functions I simply had the code in my main function again.
Possibly just having a normal while loop inside your main would achieve the same result as your two functions, without the extra work.

Using one function to call two other functions kind of just makes the code messy and hard to keep track of, as I had to keep scrolling up and down while checking each of the functions works properly.